### ISSUE TITLE
[Bug] - yaml: unmarshal errors

### ISSUE THREAD

**Issue by Marusuk3** [kasmtech/workspaces-issues#350]
### Existing Resources

- [✓] Please search the existing issues for related problems
- [✓] Consult the product documentation : [Docs](https://kasmweb.com/docs/latest/index.html)
- [✓] Consult the FAQ : [FAQ](https://kasmweb.com/docs/latest/FAQ.html)
- [✓] Consult the Troubleshooting Guide : [Guide](https://kasmweb.com/docs/latest/guide/troubleshooting.html)
- [✓] Reviewed existing training videos: [Youtube](https://www.youtube.com/@KasmWorkspaces )

**Describe the bug**
Hi, I ran into an issue when trying to download Kasm on linux-mint. I get this error: 

```
Initializing Database
Setting Seed file as /opt/kasm/1.12.0/conf/database/seed_data/default_properties.yaml
yaml: unmarshal errors:
  line 26: mapping key "restart" already defined at line 6
  line 43: mapping key "restart" already defined at line 38
kasm_db_1.12.0
yaml: unmarshal errors:
  line 26: mapping key "restart" already defined at line 6
  line 43: mapping key "restart" already defined at line 38

```

**To Reproduce**
1. sudo bash install.sh

**Expected behavior**
Install Kasm

**Screenshots**
N/A

**Workspaces Version**
Version 1.12

**Workspaces Installation Method**
Single Server

**Client Browser (please complete the following information):**
 - OS: Linux Mint

**Workspace Server Information (please provide the output of the following commands):**
N/A

Thanks

**Comment by thelamer** [comment]
This happened due to a compose release cut yesterday, you have two ways to proceed here. I modified the compose files in this installer tarball: 
https://kasm-ci.s3.amazonaws.com/kasm_release_1.12.0.d4fd8a.tar.gz

Or you can install the specific compose version we do during installation IE: (after removing your current version `sudo pacman -R docker-compose`): 
```
mkdir -p /usr/local/lib/docker/cli-plugins
curl -L \
  https://github.com/docker/compose/releases/download/v2.5.0/docker-compose-$(uname -s)-$(uname -m) -o \
  /usr/local/lib/docker/cli-plugins/docker-compose
chmod +x /usr/local/lib/docker/cli-plugins/docker-compose
```

Sorry about this and we are working to get out a patched version to be posted officially so people that have systems with Docker/Compose already installed we don't block the install due to an old typo.

**Comment by steffromfrance** [comment]
Huummm having the same problem on Ubuntu, can you explain me what I have to do to fix it ...please. Modify the .sh file ?

**Comment by thelamer** [comment]
> Huummm having the same problem on Ubuntu, can you explain me what I have to do to fix it ...please. Modify the .sh file ?

There are extra restart directives in the docker compose files contained in `kasm_release/docker/*.yaml`, the new linting in [v2.17.2](https://github.com/docker/compose/tree/v2.17.2) kicks these back and the tarball I provided above removes them. 

So as stated earlier use that tarball or pin compose to the version specified. But feel free to modify the files yourself.

**Comment by Marusuk3** [comment]
The modified installer worked fine. Thank you for your help

**Comment by CultriX-Github** [comment]
I fixed this by adding the following line at line 1165 into the install.sh file:

Before the line:
"cp ${KASM_INSTALL_BASE}/docker/.conf/docker-compose-all.yaml ${KASM_INSTALL_BASE}/docker/docker-compose.yaml"
Add the line:
sed -i ${KASM_INSTALL_BASE}/docker/.conf/docker-compose-all.yaml -e '/restart: always/d'

Now if you run the install.sh script it should work.
The error occurs because the line: "restart: always" occurs multiple time under the same service definition in the docker-compose.yaml file.  You could also manually remove it in the file: 
"${KASM_INSTALL_BASE}/docker/.conf/docker-compose-all.yaml"

Edit: Excuse me, I was too quick with my reply and noticed that @thelamer had already pointed this out in his comment...

**Comment by j-travis** [comment]
This issue was addressed in version 1.13

https://kasmweb.com/docs/latest/release_notes/1.13.0.html#bugfixes
